_includes/components/search-results/SearchResult/partials/normaliseServerResponse.js

```diff
@@ -0,0 +1,43 @@
+import { find } from 'lodash';
+import extractSnippet from './extractSnippet.js';
+
+
+const normaliseDepartmentItem = (item) => {
+  const { extras, province, financial_year: financialYear, organization = {}, title: rawTitle } = item;
+
+  const getExtrasValue = (key) => {
+    const obj = find(extras, extra => extra.key === key) || { value: null };
+    return obj.value;
+  };
+
+  const isOfficial = organization.title === 'National Treasury';
+
+  const year = financialYear[0];
+  const region = getExtrasValue('geographic_region_slug');
+  const regionString = region === 'south-africa' ? 'National' : province[0];
+  const regionSlug = region === 'south-africa' ? 'national' : `provincial/${region}`;
+
+  const nameSlug = getExtrasValue('department_name_slug');
+  const nameString = getExtrasValue('department_name');
+  const snippet = extractSnippet(item);
+
+  const title = isOfficial ?
```

Comment:
dude, if you're using ternary expressions over multiple lines just write if then else so we can read it.

Reply:
Yeah, given that was a bit lazy on my part. Maybe took the make it work a bit too far. :)